utilities/casebundle-cronjob/cronjob.yaml

```
@@ -0,0 +1,32 @@
+        spec:
+          containers:
+          - name:  CASE-BUNDLE
+            image: egovio/case-bundle
+            command: ["python3","cronJobAPIConfig.py"]
+            args: ["CASE"]  
+            env:
+              - name: TENANT_ID
+                valueFrom:
+                  configMapKeyRef:
+                    name: egov-config
+                    key: egov-state-level-tenant-id 
```

yamllint

[error] trailing spaces

(trailing-spaces)